src/components/TodoContext/TodoContext.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { createContext, useState } from 'react';
+import { Todo } from '../../types/Todo';
+
+type ContextObject = {
+  showedTodo: Todo | null;
+  setShowedTodo: (u: Todo | null) => void;
+};
```

Comment:
`u: Todo | null` what does it mean `u`? Avoid one character variables

Also, `ContextObject` isn't an informative type name, it looks like `TodoContext`, you can't use this type for other contexts